### ISSUE TITLE
Template item.php - hide if empty does not work for relation reverse field

### ISSUE THREAD

**Issue by marclight** [FLEXIcontent/flexicontent-cck#786]
I have created a template for item.php.
When I have a field that I don't want to show if empty I use these lines:

```php
<?php if (isset($item->fields["fiield_name"]->display)) : ?>
  <?php echo $item->fields["field_name"]->display; ?>
<?php endif; ?>
```

It works fine,
but it does not work when I use a **relation_reverse** field.

**Comment by ggppdk** [comment]
Fixed with 326ed6f

**Comment by ggppdk** [comment]
Please note that the issue that was fixed is not related to your issue

The relation field was producing empty when div when none of the related items was e.g. published items and show total info was ON

You code to check field is empty is incomplete, correct code is (if field is not inside fieldgroup)

```<?php if (isset($item->fields["fiield_name"]->display) && !strlen($item->fields["fiield_name"]->display)) : ?>
  <?php echo $item->fields["field_name"]->display; ?>
<?php endif; ?>
```
